packages/icons/src/ux-icon-map.ts

```diff
@@ -9,11 +12,29 @@ export interface UxIconRegObject {
 
 export type UxIconRegArray = [string, string, number?, number?];
 
+@inject(Loader)
 export class UxIconMap {
 
   public defaultIconWidth: number = 24;
   public defaultIconHeight: number = 24;
 
+  constructor(private loader: Loader) {}
+
+  public loadFullSet() {
+    PLATFORM.moduleName('./full', {chunk: 'full-icon-set'});
```

Comment:
this can just be
```ts

    PLATFORM.moduleName('./full', 'full-icon-set');
```
unless there's some specific reason for longer form syntax?

Reply:
good catch. came from previous trials with JSON loading. The longer form syntax is not necessary here.